recipes/atomgpt/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "atomgpt" %}
+{% set version = "2024.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/atomgpt/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b4620ec1b090c10fb76acc0d91342b5edee9e8de242171e02d3a04173fc8f4c2
+
+build:
+  noarch: python 
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools 
+    - wheel
+    - pip
+  run:
+    - python >=3.10
+    - numpy >=1.19.5
+    - scipy >=1.6.1
+    - jarvis-tools >=2021.07.19
+    - pytorch >=2.0.0
+    - mpmath >=1.3.0
+    # - pytorch-cuda >=12.1
```

Comment:
Is this an optional dependency? How functional is the package without it?

Reply:
these are not optional dependencies

Reply:
Then commenting it out will break the package.

Reply:
@knc6 is

```
# - pytorch-cuda >=12.1
```

A mandatory or an optional dependency?

Reply:
It's optional. I can remove it if needed

Reply:
OK. So I can disregard https://github.com/conda-forge/staged-recipes/pull/27357#discussion_r1729186899, right? No need to remove if you plan to re-add it later but please add a comment above on why you commented it out.

Reply:
Yes, please disregard that comment. I thought you are talking about jarvis-tools package.

Reply:
Can you remove the commented out dependency?

Reply:
Sure, commented it out now.